pkg/types/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/aquasecurity/tracee/pkg/types
```

Comment:
Do we really need to move it under pkg/types/? The pkg in this Go repo convention is for shared code that you can import as library. The types modules was introduced solely as a workaround to make Go plugins happy so I'd not mix it with code under pkg.

Reply:
It was an idea we had but I will move it to tracee/types